src/sagas/import-project.saga.test.js

```diff
@@ -0,0 +1,288 @@
+import { call, put, cancel, select, takeEvery } from 'redux-saga/effects';
+import rootSaga, {
+  handlePathInput,
+  showImportDialog,
+  handleImportError,
+  importProject,
+  inferProjectType,
+} from './import-project.saga';
+import {
+  importExistingProjectStart,
+  importExistingProjectError,
+  importExistingProjectFinish,
+  SHOW_IMPORT_EXISTING_PROJECT_PROMPT,
+  IMPORT_EXISTING_PROJECT_START,
+} from '../actions';
+import {
+  loadPackageJson,
+  writePackageJson,
+} from '../services/read-from-disk.service';
+import { getInternalProjectById } from '../reducers/projects.reducer';
+import { getColorForProject } from '../services/create-project.service';
+
+jest.mock('electron', () => ({
+  remote: {
+    app: { getAppPath: () => '' },
+    dialog: { showOpenDialog: jest.fn(), showErrorBox: jest.fn() },
+  },
+}));
```

Comment:
We have a shared electron mock now, in `__mocks__/electron.js`. We should add these new `dialog` methods to it, and remove this mock.